### DISCUSSION TITLE
Lesson 6, we can use interactive when we are running scripts

### DISCUSSION THREAD

**Discussion by Kaiziron** [Cyfrin/foundry-full-course-cu#455] [General]
### Lesson

Lesson 6

The video said that we can't use interactive when we are running scripts :
https://www.youtube.com/watch?v=umepbfKp5rI&t=27765s

But actually we can use interactive to enter private key when we are running scripts, instead of `--interactive`/`-i`, we can use `--interactives <num>`/`-i <num>`, and specify the sender address with `--sender`

For example :
```
forge script script/DeploySimpleStorage.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --sender 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 -i 1
```

**Comment by LordAdhaar** [comment]
use --interactives 1
and enter the private key of your account to deploy the script.

**Comment by EngrPips** [comment]
Interesting. I will try that out @Kaiziron.
